src/main/java/com/tiff/tffnserachservice/controller/TiffnerController.java

```diff
@@ -0,0 +1,64 @@
+package com.tiff.tffnserachservice.controller;
+
+
+import com.tiff.tffnserachservice.dto.TiffnerDTO;
+import com.tiff.tffnserachservice.model.Tiffner;
+import com.tiff.tffnserachservice.repository.TiffnerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/tffn-search")
+public class TiffnerController {
+
+    @Autowired
+    private TiffnerRepository tiffnerRepository;
+
+    @GetMapping
+    public List<Tiffner> findTiffns() {
+        return tiffnerRepository.findAll();
+    }
+
+    @GetMapping("/api/find-tiffn/{id}")
```

Comment:
lets remove the /api part from here for now because that will make our url look like this:
 `<application_host><port>/tffn-search/api/find-tiffin/1`

lets just keep it ; tffn-search/tffns/1
we should refrain from using find-tiffin type names for our service names, we want nouns and not verbs afaik
 good reference https://restfulapi.net/resource-naming/
 
 
-never use CRUD function naming in the URIs